Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/transform/subdisc/HL7Extractors.java

```diff
@@ -316,12 +317,12 @@ public static PersonNameType translateENListtoPersonNameType(List<ENExplicit> na
         LOG.debug("HL7Extractor.translateENListtoPersonNameType() -- Begin");
         PersonNameType personName = new PersonNameType();
         // NameType
-        if (names.size() > 0 && names.get(0).getUse().size() > 0) {
+        if (CollectionUtils.isNotEmpty(names) && CollectionUtils.isNotEmpty(names.get(0).getUse())) {
             CeType nameType = new CeType();
             nameType.setCode(names.get(0).getUse().get(0));
             personName.setNameType(nameType);
         }
-        if (names.size() > 0 && names.get(0).getContent() != null) {
+        if (CollectionUtils.isNotEmpty(names) && names.get(0).getContent() != null) {
```

Comment:
You are right Not null and empty checks on a collection are not same. However when we look at this method, getting an iterator and doing a while iterator.hasnext(), yields the same purpose as having an empty check in the if condition.